plugins/network/commands

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
```

Comment:
Yeah, just wanted to write the integration as a proof of concept before I got down and dirty with golang.